_sources/structures/misc/steeringmanager.txt

```diff
@@ -0,0 +1,233 @@
+
+.. attribute:: SteeringManager:PITCHTS
+
+    :type: scalar
+    :access: Get/Set
+
+    Represents the settling time for the PID calculating pitch torque based on target angular velocity.  The proportional and integral gain is calculated based on the settling time and the moment of inertia in the pitch direction.  Ki = (moment of inertia) * (4 / (settling time)) ^ 2.  Kp = 2 * sqrt((moment of inertia) * Ki).
+
+.. attribute:: SteeringManager:YAWTS
+
+    :type: scalar
+    :access: Get/Set
+
+    Represents the settling time for the PID calculating yaw torque based on target angular velocity.  The proportional and integral gain is calculated based on the settling time and the moment of inertia in the yaw direction.  Ki = (moment of inertia) * (4 / (settling time)) ^ 2.  Kp = 2 * sqrt((moment of inertia) * Ki).
+
+.. attribute:: SteeringManager:ROLLTS
+
+    :type: scalar
+    :access: Get/Set
+
+    Represents the settling time for the PID calculating roll torque based on target angular velocity.  The proportional and integral gain is calculated based on the settling time and the moment of inertia in the roll direction.  Ki = (moment of inertia) * (4 / (settling time)) ^ 2.  Kp = 2 * sqrt((moment of inertia) * Ki).
+
+.. attribute:: SteeringManager:MAXSTOPPINGTIME
+
+    :type: scalar (s)
+    :access: Get/Set
+
+    This value is used to limit the turning rate when calculating target angular velocity.  The ship will not turn faster than what it can stop in this amount of time.  The maximum angular velocity about each axis is calculated as: (max angular velocity) = MAXSTOPPINGTIME * (available torque) / (moment of inertia).
+
+.. attribute:: SteeringManager:ANGLEERROR
+
+    :type: scalar (deg)
+    :access: Get only
+
+    The angle between the ship's facing direction forward vector and the target direction's forward.  This is the combined pitch and yaw error.
+
+.. attribute:: SteeringManager:PITCHERROR
+
+    :type: scalar (deg)
+    :access: Get only
+
+    The pitch angle between the ship's facing direction and the target direction.
+
+.. attribute:: SteeringManager:YAWERROR
+
+    :type: scalar (deg)
+    :access: Get only
+
+    The yaw angle between the ship's facing direction and the target direction.
+
+.. attribute:: SteeringManager:ROLLERROR
+
+    :type: scalar (deg)
+    :access: Get only
+
+    The roll angle between the ship's facing direction and the target direction.
+
+.. attribute:: SteeringManager:PITCHTORQUEADJUST
+
+    :type: scalar (kNm)
+    :access: Get/Set
+
+    You can set this value to provide an additive bias to the calculated available pitch torque. (available torque) = ((calculated torque) + PITCHTORQUEADJUST * PITCHTORQUEFACTOR.
```

Comment:
There's some missing parentheses in most, but not all, of these `*TORQUEADJUST` and `*TORQUEFACTOR` descriptions.

i.e.

`(available torque) = ((calculated torque) + PITCHTORQUEADJUST * PITCHTORQUEFACTOR.`
`.............................................................^`